frontend/src/config/bhutan/prism.json

```diff
@@ -0,0 +1,188 @@
+{
+  "country": "Bhutan",
+  "defaultLanguage": "en",
+  "countryAdmin0Id": 31,
+  "alertFormActive": false,
+  "hidePanel": false,
+  "enableNavigationDropdown": true,
+  "icons": {
```

Comment:
Same, let's rely on the shared config as much as possible for things like that. Don't hesitate to add to the shared/prism.json